WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/network/JsonTransferer.java

```diff
@@ -38,10 +39,11 @@ public JsonTransferer() {
 	}
 
 	/**
-	 * Sends the recorded T's to the server. Returns whether or not the transfer
+	 * Sends the recorded items to the server. Returns whether or not the transfer
```

Comment:
I see why you went for the name item as an abstraction for Interval + Event, however, I find it too generic and it does not really tell anything about the underlying nature. Moreover, the concept of items does not exist in the code (for example, as a class). A few suggestions: WatchDogTransferable (maybe now is the time to rename that to something different?), WatchDogItem, IDEActivity